### PR TITLE
Added settings to choose multitasking preferences

### PR DESCRIPTION
**Pull request by lukeblevins**
We add a new page in Settings for users to opt out of the default adaptive multitasking behavior:
image

### CODE REVIEW

File: Files/App.xaml.cs [hunk 1]
```diff
@@ -34,11 +34,14 @@
 using System.Linq;
 using Newtonsoft.Json;
 using Files.Common;
+using Files.UserControls.MultiTaskingControl;
 
 namespace Files
 {
     sealed partial class App : Application
     {
+        public static IMultitaskingControl multitaskingControl = null;
```
Comment (tsvietOK) [inline Files/App.xaml.cs:43]:
```suggestion
        public static IMultitaskingControl MultitaskingControl = null;
```

**Comment by duke7553** [comment]
Some of the fixes from newer versions of Files aren't in this branch, so there are some bugs present which will be fixed when this is merged.

**Comment by duke7553** [comment]
I also found a crash which occurs on resizing the app window. It only occurs in a debugger though. Can't repro elsewhere. If anyone has any ideas please let me know.

File: Files/View Models/SettingsViewModel.cs [hunk 2]
```diff
@@ -434,16 +436,58 @@ public bool AreLinuxFilesSupported
...
+                Set(value);
+                if (value)
+                {
+                    Window.Current.SizeChanged += Current_SizeChanged;
+                    if (App.InteractionViewModel.IsWindowCompactSize)
+                    {
+                        App.InteractionViewModel.IsVerticalTabFlyoutVisible = true;
+                        App.InteractionViewModel.IsHorizontalTabStripVisible = false;
+                    }
+                    else if (!App.InteractionViewModel.IsWindowCompactSize)
```
Comment (tsvietOK) [inline Files/View Models/SettingsViewModel.cs:453]:
```suggestion
                    else
```

**Review by tsvietOK — DISMISSED** [review]

**Comment by Jaiganeshkumaran** [comment]
@duke7553 It might be a XAML issue because I have seen exceptions with XAML thrown when running inside the debugger.

**Review by yair100 — DISMISSED** [review]

**Comment by tsvietOK** [comment]
@yair100 Changing the type of multitasking layout from horizontal to vertical does not change it.

**Comment by yair100** [comment]
> @yair100 Changing the type of multitasking layout from horizontal to vertical does not change it.

I will take a look.

**Comment by yair100** [comment]
> @yair100 Changing the type of multitasking layout from horizontal to vertical does not change it.

I cannot reproduce this issue.

**Review by yair100 — APPROVED** [review]

**Pull request merged**